### PR TITLE
PMM-6897 Add suspend/resume psmdb

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-6897

ref:
https://github.com/percona-platform/dbaas-controller/pull/100
https://github.com/percona-platform/dbaas-api/pull/69

### CODE REVIEW

**Review by AlekSi — CHANGES_REQUESTED** [review]
Please change the base branch

**Review by BupycHuk — CHANGES_REQUESTED** [review]
The same comments as for PXC

**Review by percona-csalguero — APPROVED** [review]

File: api/managementpb/dbaas/psmdb_cluster.proto [hunk 1]
```diff
@@ -70,12 +72,27 @@ message ListPSMDBClustersResponse {
...
     RunningOperation operation = 3;
     // Cluster parameters.
-    PSMDBClusterParams params = 4;
+    PSMDBClusterParamsList params = 4;
   }
   // PSMDB clusters information.
   repeated Cluster clusters = 1;
 }
 
+// PSMDBClusterParamsList represents PSMDB cluster parameters that can be listed.
```
Comment (BupycHuk) [inline api/managementpb/dbaas/psmdb_cluster.proto:81]: As far as I see this message doesn't have difference from `PSMDBClusterParams`

Comment (BupycHuk) [reply api/managementpb/dbaas/psmdb_cluster.proto:81]: So maybe we can revert this changes.

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: api/managementpb/dbaas/psmdb_cluster.proto (hunk 1 above)
Comment (askomorokhov) [reply api/managementpb/dbaas/psmdb_cluster.proto:81]: Yes, right - wil do.

Comment (askomorokhov) [reply api/managementpb/dbaas/psmdb_cluster.proto]: Reverted.

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**